numalogic/tools/data.py

```diff
@@ -70,21 +118,38 @@ def __init__(
         self.val_dataset = None
 
     def setup(self, stage: str) -> None:
+        """
+        Sets up the data module by initializing the train and validation datasets.
+        """
         if stage == "fit":
             self.train_dataset = StreamingDataset(self.train_data, self.seq_len)
             if self.val_data is None:
                 return
             self.val_dataset = StreamingDataset(self.val_data, self.seq_len)
 
     def train_dataloader(self) -> TRAIN_DATALOADERS:
+        """
```

Comment:
nitpick: any reason that we use r""" in some places but """ in others?

Reply:
Thanks probably missed it, will make it consistent.